docs/system_setup.rst

```diff
@@ -0,0 +1,390 @@
+    # Where can GRDC observation files (<station identifier>_Q_Day.Cmd.txt) be found.
+    ewatercycle.CFG['grdc_location'] = './grdc-observations'
+    # Where can parameters sets prepared by the system administator be found
+    ewatercycle.CFG['parameterset_dir'] = './parameter-sets'
+    # Where is the configuration saved or loaded from
+    ewatercycle.CFG['ewatercycle_config'] = './ewatercycle.yaml'
+
+and then written to disk with
+
+.. code:: ipython3
+
+    ewatercycle.CFG.save_to_file()
+
+Later it can be loaded by using:
+
+.. code:: ipython3
+
+    ewatercycle.CFG.load_from_file('./ewatercycle.yaml')
+
+To make the ewatercycle configuration load by default for current user
+it should be copied to ``~/.config/ewatercycle/ewatercycle.yaml`` .
+
+To make the ewatercycle configuration available to all users on the
+system it should be copied to ``/etc/ewatercycle.yaml`` .
+
+Configuration file for Cartesius system
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Users part of the ewatercycle project on the Cartesius system of
+SURFSara can use the following configuration file:
+
+.. code:: yaml
+
+   container_engine: singularity
+   singularity_dir: /projects/0/wtrcycle/singularity-images
+   output_dir: /scratch/shared/ewatercycle
+   grdc_location: /projects/0/wtrcycle/GRDC
+   parameterset_dir: /projects/0/wtrcycle/parameter-sets
+
+Configuration file for ewatecycle Jupyter machine
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+On systems constructed with ewatercycle application on SURF Research
+Cloud can use the following configuration file:
+
+.. code:: yaml
+
+   container_engine: singularity
+   singularity_dir: /mnt/data/singularity-images
+   output_dir: /scratch
+   grdc_location: /mnt/data/GRDC
+   parameterset_dir: /mnt/data/parameter-sets
+
+Model container images
+----------------------
+
+As hydrological models run in container their container images should be
+made available on the system.
+
+The names of the images can be found in the ``ewatercycle.models.*``
+classes.
+
+Docker
+~~~~~~
+
+Docker images will be downloaded with ``docker pull``:
+
+.. code:: shell
+
+    docker pull ewatercycle/lisflood-grpc4bmi:20.10
+    docker pull ewatercycle/marrmot-grpc4bmi:2020.11
+    docker pull ewatercycle/pcrg-grpc4bmi:setters
+    docker pull ewatercycle/wflow-grpc4bmi:2020.1.1
+    # TODO
+
+Singularity
+~~~~~~~~~~~
+
+Singularity images should be stored in configured directory
+(``ewatercycle.CFG['singularity_dir']``) and can build from Docker with:
+
+.. code:: ipython3
+
+    !cd {ewatercycle.CFG['singularity_dir']}
+    !singularity build ewatercycle-lisflood-grpc4bmi.sif docker://ewatercycle/lisflood-grpc4bmi:latest
+    !singularity build ewatercycle-pcrg-grpc4bmi.sif docker://ewatercycle/pcrg-grpc4bmi:latest
+    !singularity build ewatercycle-marrmot-grpc4bmi.sif docker://ewatercycle/marrmot-grpc4bmi:latest
+    !singularity build ewatercycle-wflow-grpc4bmi.sif docker://ewatercycle/wflow-grpc4bmi:latest
```

Comment:
Shall we use the model versions as the tag instead of `latest`?

Reply:
Now using same labels as used in model classes